Documentation/Upgrade/rook-upgrade.md

```diff
@@ -47,22 +48,17 @@ those releases.
     official releases. Builds from the master branch can have functionality changed or removed at any
     time without compatibility support and without prior notice.
 
-## Breaking changes in v1.13
-
-* The minimum supported version of Kubernetes is v1.23.
-  Upgrade to Kubernetes v1.23 or higher before upgrading Rook.
-* The minimum supported version of Ceph is v17.2.0.
-  If a lower version is currently deployed, [Upgrade Ceph](./ceph-upgrade.md) before upgrading Rook.
-* CephCSI CephFS driver introduced a breaking change in v3.9.0. If any existing CephFS storageclass in
-  the cluster has `MountOptions` parameter set, follow the steps mentioned in the
-  [CephCSI upgrade guide](https://github.com/ceph/ceph-csi/blob/v3.9.0/docs/ceph-csi-upgrade.md/#upgrading-cephfs)
-  to ensure a smooth upgrade.
-  This became the default CSI version in Rook v1.12.1, and may have already been resolved.
-* Support for the admission controller has been removed. CRD validation is now enabled with
-  [Validating Admission Policies](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/).
-  Validating Admission Policy rules are ignored in Kubernetes v1.24 and lower.
-  If the admission controller is enabled, it is advised to upgrade to Kubernetes v1.25 or higher before upgrading Rook.
-  For more info, see https://github.com/rook/rook/pull/11532.
+## Breaking changes in v1.14
+
+* The minimum supported version of Kubernetes is v1.25.
+  Upgrade to Kubernetes v1.25 or higher before upgrading Rook.
+* The Rook operator config `CSI_ENABLE_READ_AFFINITY` was removed. v1.13 clusters that have modified
+  this value to be `"true"` must set the option as desired in each CephCluster as documented
+  [here](https://rook.github.io/docs/rook/v1.14/CRDs/Cluster/ceph-cluster-crd/#csi-driver-options)
+  before upgrading to v1.14.
+* Rook is beginning the process of deprecating CSI network "holder" pods.
+  Helm chart users should set the new config `disableHolderPods: false` before upgrading to v1.14.
```

Comment:
This only applies to users who have `networkProvider: multus`, right? I thought no action would be required by anyone else since they wouldn't have the holder pods in their clusters.

Reply:
This can also apply to non-multus users who use CSI without host network.

Reply:
Ok, so two conditions: `networkProvider: multus` or `CSI_ENABLE_HOST_NETWORK: "false"`? 

Neither of those are the default so hopefully few users actually need to go through the process to disable the hold pods. I'm hoping if we're specific here about who is affect, unaffected users won't need to jump to the other doc. For example, this wording here sounds like all helm users are affected.

Reply:
It seems easiest to tell all users to set the value so we don't have any affected users miss it. But you do have a point. Do you have suggested wording to keep this short and sweet as a middle ground?

Reply:
Perhaps we tell them to look for these pods to know if they are affected, similar to your hint in the doc.
```suggestion
  If there are pods named "*plugin-holder*" in your rook namespace, see the detailed documentation to disable them.
```

Reply:
Okay. I changed the wording in several sections (including this one) based on the suggestion.